src/lvl_script_commands.c

```diff
@@ -3597,6 +3602,214 @@ static void set_creature_configuration_process(struct ScriptContext* context)
         case 36: // TORTUREKIND
             crstat->torture_kind = value;
             break;
+        case 37: // SPELLIMMUNITY
+            switch (value)
```

Comment:
You can avoid this huge switch. You can simply do `set_flag(crstat->immunity_flags, value);`

Reply:
Then put the enum values in the NamedCommand too.

Reply:
or a 1>>value is also an option without changing the NamedCommand

Reply:
> > Then put the enum values in the NamedCommand too.
> 
> Are you sure it's wise to do that? 🤔

It's being used in other places, but if what you and qq cooked up works, fine by me.